gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckUnusedDependenciesTask.java

```diff
@@ -176,7 +176,7 @@ private boolean shouldIgnore(ResolvedArtifact artifact) {
         return ignore.get().contains(BaselineExactDependencies.asString(artifact));
     }
 
-    @Input
+    @InputFiles
```

Comment:
this is really the key, if you don't set this then gradle doesn't wire up the tasks that produce various files inside these configurations as inputs...
I think we were just getting lucky before.